### PR TITLE
Fix Modal docs for isOpen

### PR DESCRIPTION
**Pull request by jtanner**

### CODE REVIEW

**Comment by lukeschunk** [comment]
:shipit:

**Comment by cerinman** [comment]
The `Modal` component doesn't take an `isOpen` prop anymore.  That line should be removed and the Modal wrapped in a ternary using a state variable to determine if it's rendered or not.

```javascript
{this.state.isOpen ? (<Modal />) : null}
```

or 

```javascript
{this.state.isOpen && <Modal />}
```

**Comment by lukeschunk** [comment]
i totally should have caught that

**Comment by cerinman** [comment]
So what's the deal with this?  Are you going to update and remove the prop or here or close this PR and start a another?

**Comment by jtanner** [comment]
@cerinman I've been meaning to come back to this and review the current docs with how the component actually works, but I haven't made time yet. I'll make some time to do it soon.

File: components/ModalDocs.js [hunk 1]
```diff
@@ -152,39 +158,40 @@ const ModalDocs = React.createClass({
...
-      isOpen={this.state.showModal}
-      onRequestClose={_handleModalClose}
-      showFooter={true}
-      showTitleBar={true}
-      tooltipLabel='This is the footer text.'
-      title='This is the header text'
-      tooltip='This is my tooltip content'
-      tooltipTitle='This is my tooltip title'
-    >
+          <Modal
```
Comment (jtanner) [inline components/ModalDocs.js]: I copied the code for the `Show Default Modal` button and put it here as the example.

**Comment by jtanner** [comment]
@cerinman @lukeschunk This is ready for another look. I updated the example and added a couple missing props.

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**